models/commentsModel.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../db/connection.js");
+
+exports.selectComments = (review_id) => {
```

Comment:
I think you should handle this with PSQL error handling - for the bad request - as PSQL will kick an error if the datatype isn't valid. Try and write this with a single db.query for the review_id.